### PR TITLE
Rel 0.1.6: Option 5

### PR DESCRIPTION
**Pull request by FabrizioMoggio**
according to: https://github.com/camaraproject/CallForwardingSignal/discussions/8#discussioncomment-9325822

#### What type of PR is this?
* enhancement/feature

### CODE REVIEW

File: code/API_definitions/Call_Forwarding_Signal.yaml [hunk 1]
```diff
@@ -49,10 +51,21 @@ info:
...
-    This Resource contains the status of the Network Call Forwarding service for
-    the specified phone number.
+    This is the resource the API Consumer gets back, containing the information
+    about the general status of the Network Call Forwarding service for the
+    specified phone number.\
+    \
+    The CFS API provides two endpoints fulfilling the following intents:
+    - **unconditional-call-forwardings**: Is the unconditional call fwd service
+    active on a specific phone number?
+    - **call-forwardings**: Provide me the status of the call FWD Service on a
```
Comment (bigludo7) [inline code/API_definitions/Call_Forwarding_Signal.yaml:67]: As we use interrogative form for unconditional-call-forwardings we should do same for call-forwardings for consistency. "What is the status of the call forwarding for a specific phone number?"

**Review by bigludo7 — APPROVED** [review]
Look good for me with 2 small comments.

File: code/API_definitions/Call_Forwarding_Signal.yaml (hunk 1 above)
Comment (FabrizioMoggio) [reply code/API_definitions/Call_Forwarding_Signal.yaml:67]: I had the same doubt myself.

I have just modified it: https://github.com/camaraproject/CallForwardingSignal/discussions/6#discussioncomment-9327549

Comment (FabrizioMoggio) [reply code/API_definitions/Call_Forwarding_Signal.yaml:67]: maybe "Which is the...." ?

Comment (bigludo7) [reply code/API_definitions/Call_Forwarding_Signal.yaml:67]: Yeees ;)

**Review by bigludo7 — APPROVED** [review]

**Pull request merged**